SDL/Triplet.h

```diff
@@ -86,9 +81,10 @@ namespace SDL {
     Buf<TDev, unsigned int> nMemoryLocations_buf;
     Buf<TDev, uint8_t> logicalLayers_buf;
     Buf<TDev, unsigned int> hitIndices_buf;
-    Buf<TDev, FPX> betaIn_buf;
-    Buf<TDev, FPX> betaOut_buf;
-    Buf<TDev, FPX> pt_beta_buf;
+    Buf<TDev, float> betaIn_buf;
```

Comment:
this should stay FPX
```suggestion
    Buf<TDev, FPX> betaIn_buf;
```